tests/example_app/example/about/templates/about.html

```diff
@@ -1,10 +1,8 @@
 {% extends "layout.html" %}
 
 {% block body %}
-  <br/>
-  <br/>
-  <a href="/about/our-story">Our Story</a>
-  <a href="/about/our-team">Our Team</a>
+  <a href="{{ url_for('about.story.story') }}">Our Story</a>
```

Comment:
Can you rename both bp paths (story and team) to reference the new `.index` rename?